x-pack/plugins/apm/kibana.json

```diff
@@ -2,7 +2,7 @@
   "id": "apm",
   "owner": {
```

Comment:
Off topic: I love that we have these plugin manifests. Would it make sense to add a `test` section, thereby making it much easier to get an overview and run all tests for a specific solution:

```json
{
  "id": "apm",
  "test": {
    "unit": "yarn jest",
    "ftr:cypress": "node scripts/ftr_e2e/cypress_run.js",
    "ftr:api:trial": "x-pack/test/apm_api_integration/trial/config.ts",
    "ftr:api:basic": "x-pack/test/apm_api_integration/basic/config.ts",
    "ftr:functional": "x-pack/test/functional/config.js",
  }
}
```

To run all APM's tests we could have a command like: `yarn test apm`. To run a single of APM's tests: `yarn test apm ftr:api:basic`

Reply:
It's an interesting idea for sure. We're getting closer to moving the tests out of the root x-pack/test dir and into the plugins, which might make this a little less helpful but it's an interesting project to think through.